sklearn/_config.py

```diff
@@ -41,6 +41,13 @@ def get_config():
     --------
     config_context : Context manager for global scikit-learn configuration.
     set_config : Set global scikit-learn configuration.
+
+    Examples
+    --------
+    >>> import sklearn
+    >>> config = sklearn.get_config()
+    >>> config['assume_finite'],config['working_memory'],config['assume_finite']
+    (False, 1024, False)
```

Comment:
Just in case that we change the default in the future, let's be lenient.

```suggestion
    >>> import sklearn
    >>> config = sklearn.get_config()
    >>> config.keys()
    dict_keys([...])
```